pyrasterframes/src/main/python/pyrasterframes/rf_types.py

```diff
@@ -344,10 +344,16 @@ def __matmul__(self, right):
         return Tile(np.matmul(self.cells, other))
 
     def dimensions(self):
-        # list of cols, rows as is conventional in GeoTrellis and RasterFrames
+        """ Return a list of cols, rows as is conventional in GeoTrellis and RasterFrames."""
         return [self.cells.shape[1], self.cells.shape[0]]
 
 
+    def _repr_png_(self):
```

Comment:
I like this solution a lot more.

Did you want to reinstate `_repr_html_` since you have the code for it?

Reply:
I chose not to because I am not sure how IPython decides which `_repr_*_` to use, and the html flavor is always going to be more expensive.